package.json

```diff
@@ -13,7 +13,12 @@
     "type": "git",
     "url": "git+https://github.com/CanopyTax/react-disposable-decorator.git"
   },
-  "author": "Keith Halterman & Joel Denning",
+  "contributors": [
+    {"name": "Keith Halterman"},
+    {"name": "Joel Denning"},
+    {"name": "Dan Kelser"},
+    {"name": "Justin McMurdie"}
+  ],
```

Comment:
I updated this because I noticed this wasn't "Correct" package.json according to [this](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) while I was looking into how peerDependencies worked